onnxruntime/core/session/inference_session.cc

```diff
@@ -96,6 +96,19 @@ inline std::basic_string<T> GetCurrentTimeString() {
 
 std::atomic<uint32_t> InferenceSession::global_session_id_{1};
 
+// The current model versions for saving the ort format models
+// This version is NOT onnxruntime version
+// Only update this version when there is a file format change which will break the compatibilites
+// Once this model version is updated, the s_supported_ort_model_versions also need to be updated
+static constexpr const char* s_ort_model_version = "1.4.0";
+
+// The ort model versions we will support in this version of onnxruntime
+// This may contain more versions than the s_ort_model_version, based on the compatibilities
+// TODO, only include this in ENABLE_ORT_FORMAT_LOAD
+static const std::unordered_set<std::string> s_supported_ort_model_versions{
+    std::string(s_ort_model_version),
+};
```

Comment:
May work better long term to have a map of ORT version to schema version instead of just a set of supported versions. Lookup schema version for model version, and decide if the schema version is supported.

Reply:
The problem is that, say if we did not update the schema at all in 1.6.0, the model won't be supported in 1.5.0 unless we come back and update the code